wiki/snapshot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Snapshots
+layout: wiki
+---
+
+# Overview
+Snapshots are a snapshot of the chain at a particular blockheight.
+Gridcoin snapshots are created by trusted developers and they allow newer users to get into sync faster than syncing from 0.
+
+# Issues
+* Snapshots are not validated by the wallet.
+* Snapshots are an inherently centralised method of getting into sync.
+* Users may mistakenly install a malicious or corrupted snapshot. 
+If enough users install the bad snapshot then the integrity of the chain could be compromised.
+
+# Using snapshots
+## Starting from 5.0.0
+Snapshots can now be downloaded and applied automatically by the wallet.
+
+It is accessible from File -> Snapshot download
+
+## Manual method
+1. [Download the snapshot](https://snapshot.gridcoin.us/snapshot.zip)
+2. [Verify the checksum (IMPORTANT)](https://snapshot.gridcoin.us/snapshot.zip.sha256)
+3. Delete the following folders: `txleveldb` & `accrual` from your Gridcoin data directory.
+4. Delete the `blk****.dat` files from your Gridcoin data directory.
+5. Extract the snapshot zip into your Gridcoin data directory.
+6. Restart the wallet and get into sync.
+
+# Alternatives
+* Syncing from 0:
+  
+  This is preferred. With all of the recent changes in the wallet it is only mildly slower than using a snapshot.
+  This doesn't require trust in the snapshot creator.
+* Bootstrap:
+  
+  As of April 2021 these are no longer offered.
+  It doesn't bypass the checks in the wallet which prevent malicious bootstraps from damaging the chain.
+  There is little to no point in using one of these over syncing from 0. 
+  It doesn't inherently bring any speed benefits compared to syncing normally but requires on a centralised download.
```

Comment:
I believe it can technically help speed things up, but yeah not by much
```suggestion
  It brings limited speed benefits compared to syncing normally but requires on a centralised download.
```